src/Commands.ts

```diff
@@ -0,0 +1,8 @@
+import { Command } from './Command';
+import { FxTweet } from './commands/FxTweet';
+import { Hello } from './commands/Hello';
+
+export const Commands: Command[] = [
+    Hello, 
+    FxTweet
+];
```

Comment:
Nit: type may be inferred
```suggestion
export const Commands = [
    Hello, 
    FxTweet
];
```